src/libs/ajax/data-table-providers/WdsDataTableProvider.ts

```diff
@@ -56,7 +64,24 @@ export class WdsDataTableProvider implements DataTableProvider {
     supportsTypeRenaming: false,
     supportsExport: false,
     supportsPointCorrection: false,
-    supportsFiltering: false
+    supportsFiltering: false,
+    supportsTabBar: false
+  }
+
+  tsvFeatures: TSVFeatures = {
+    needsTypeInput: true,
+    uploadInstructions: 'Choose the data to import below. ',
+    sampleTSVLink: 'src/../wds_template.tsv', //TODO: placeholder, does not currently work
```

Comment:
Just flagging here -- do you need to edit this placeholder? Or rather remove this `TODO` comment?

Reply:
I was hoping that we'd have a place to put it on Azure before this PR was ready to merge, so I'm going to hold out a bit longer.

Reply:
Oh I missed this comment and put the template in a bucket and fixed the link.  Presumably it doesn't need to be in the repository, though, since the Entity Service template isn't - although that seems to assume it will never be changed.